src/controllers/country.controller.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Response, Request } from 'express';
+import * as contryService from '../services/country.service';
+import { getPhoneNumberByCountry } from '../services/phoneNumber.service';
+
+export const getPhoneNumbers = async(req: Request, res: Response) => {
+  try {
+    const country_code = req.params.code;
+
+    if(country_code){
```

Comment:
convention của  hình như à có space á c
```suggestion
    if (country_code) {
```